src/main/java/com/minecolonies/entity/ai/basic/AbstractAISkeleton.java

```diff
@@ -213,4 +224,9 @@ public final AIState getState()
     {
         return state;
     }
+
+    protected int getLevelDelay()
+    {
+        return 10;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS3400)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 10 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)